### PR TITLE
Add documentation for new `is` operator

### PR DESCRIPTION
**Pull request by john-h-kastner-aws**
Adding docs for `is` operator  which will be released in the upcoming Cedar 3.0 release. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/collections/_policies/syntax-grammar.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ For more details, see [`&&` \(AND\)](../policies/syntax-operators.html#operator-
 ## `Relation` {#grammar-relation}
 
 ```
-Relation ::= Add [RELOP Add] | Add 'has' (IDENT | STR) | Add 'like' PAT
+Relation ::= Add [RELOP Add] | Add 'has' (IDENT | STR) | Add 'like' PAT | Add 'is' Path ('in' Add)?
```
Comment (khieta) [inline docs/collections/_policies/syntax-grammar.md:120]: Also need to add `is` to scope rules `Principal` and `Resource` above.

**Review by khieta — COMMENTED** [review]
Overall content looks good, although I'll leave the final approval to @AMZ-brandon.

Please also update docs/collections/_other/doc-history.md to reflect that the `is` operator corresponds to a new language version (2.2.0?).

**Comment by cdisselkoen** [comment]
> to reflect that the is operator corresponds to a new language version (2.2.0?).

Cedar SDK 3.0 will also have to be language 3.0; although `is` is not a breaking change for the language, things like RFC 20 are.

**Comment by AMZ-brandon** [comment]
I'm putting together a preview in my test repo. This branch is behind main - can you update it from the base branch?

**Comment by john-h-kastner-aws** [comment]
updated

**Comment by AMZ-brandon** [comment]
Thanks for updating! I've pulled the branch - preview [here](https://amz-brandon.github.io/).

I am updating the document history and adding [syntax highlighting](https://github.com/cedar-policy/cedar-docs/blob/main/docs/js/hljs-cedar.min.js) to `is`.

What should be the new language version for the [doc history](https://amz-brandon.github.io/other/doc-history.html)? 3.0? And what PR in the Cedar repo matches this update?

**Comment by john-h-kastner-aws** [comment]
> What should be the new language version for the [doc history](https://amz-brandon.github.io/other/doc-history.html)? 3.0?

Yes. This will be released with version 3.0 (to be release soon).

> And what PR in the Cedar repo matches this update?

`is` was added in https://github.com/cedar-policy/cedar/pull/396 following to the design in [rfc5](https://github.com/cedar-policy/rfcs/blob/main/text/0005-is-operator.md)

**Comment by john-h-kastner-aws** [comment]
We should wait on the merge until the 3.0 release happens so we don't have `is` referenced in the docs before the new version is on crates.io. That will be a week or two probably.

**Comment by AMZ-brandon** [comment]
Got it, standing by until I hear of the release.

**Pull request merged**